Jenkinsfile

```diff
@@ -18,6 +18,24 @@ pipeline {
     parameters {
         string(name: 'PAVICS_HOST', defaultValue: default_pavics_host,
                description: 'PAVICS host to run notebooks against.', trim: true)
+        // TEST_MAGPIE_AUTH enables the evaluation of end-2-end access to some secured resources by Twitcher/Magpie
+        // and validate that resulting resources are found (or blocked) after proxy resolution of granted/denied access.
+        // If Authentication/Authorization is not needed or not employed for your instance, this should be disabled to
+        // avoid failures.
+        // NOTE:
+        //   This test suite might require manual clean-up on failure (if critical error).
```

Comment:
Can you put this line "This test suite might require manual clean-up on failure (if critical error)" into the description of the checkbox itself.  Jenkins user won't see comments in the Jenkinsfile.